_source/_posts/2024-02-29-3pc.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: blog_post
+title: "The End of Third-Party Cookies"
+author: edunham
+by: advocate
+communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
+description: ""
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image: blog/3pc/social.jpg
+type: awareness
+---
+
+## What are third-party cookies?
+
+Cookies are as old as the internet. Historically, cookies were among the only options for personalizing a user's online experience and carrying their preferences from page to page. First-party cookies are issued by the web site where they're used, and third-party cookies come from other domains. 
+
+third-party cookies allow user behavior to be tracked across different sites which they visit. These cookies are now creatively misused by advertisers and trackers to exploit personal information. For the legitimate tasks which used to require third-party cookies, like social integration and caching, there are now better and more privacy-respecting technologies available. 
+
+Today, the drawbacks to users' security and privacy from third-party cookie implementations outweigh their benefits so much that all major browsers are phasing them out. Firefox, Edge, and Safari have been blocking third-party cookies for years, and Chrome is now phasing them out in 2024. 
+
+If your code uses Okta features that rely on third-party cookies, this means that you'll need to make some changes to keep the identity experience working as intended. 
+
+
+## Does your application use third-party cookies? 
+
+Most code using Okta doesn't use third-party cookies, and won't be affected by the deprecation. Here are the design patterns in which Okta uses third-party cookies. If your application is in one of these categories, please test its behavior with third-party cookie deprecation. 
```

Comment:
> Most code using Okta doesn't use third-party cookies, and won't be affected by the deprecation.

From a purely technical perspective, I am not sure this is accurate. There are some cases where the cookie deprecate may "degrade" a user's experience, but still function

For example, "Remember Me" uses a cookie to remember if a user preformed MFA 2nd factor auth already. Without this cookie, the user will always be prompted for MFA. It doesn't block auth, but does affect the user experience

Reply:
Rewrote this section for better accuracy :)